blueprints.yaml

```diff
@@ -287,3 +287,33 @@ form:
               label: PLUGINS.JS_COMMENTS.PROVIDERS.MUUT.FIELDS.LANGUAGE
               validate:
                 type: string
+
+        hypercomments:
+          type: tab
+          title: PLUGINS.JS_COMMENTS.PROVIDERS.HYPERCOMMENTS.TITLE
+
+          fields:
+            providers.hypercomments.widget_id:
+              type: text
+              label: PLUGINS.JS_COMMENTS.PROVIDERS.HYPERCOMMENTS.FIELDS.WIDGET_ID
+              size: x-small
+              validate:
+                type: int
+
+#            providers.hypercomments.show_count:
```

Comment:
Can you explain why the comments counts are commented out? Does HyperComments in principle supports comments counts? Did you test it?